lib/superstore/types/array_type.rb

```diff
@@ -1,8 +1,12 @@
 module Superstore
   module Types
     class ArrayType < Base
+      def serialize(value)
+        value if value.present?
```

Comment:
`super if value.present?` also works here, but the override paradigm [elsewhere](https://github.com/search?q=repo%3Adata-axle%2Fsuperstore%20def%20serialize&type=code) is to just return `value`.